google-cloud-bigtable-stats/pom.xml

```diff
@@ -0,0 +1,89 @@
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- This module is experimental. It is a workaround to support and publish built-in metrics to cloud monitoring
+    through Stackdriver. Built-in metrics will be implemented with shaded OpenCensus so it won't interfere with
+    customer's application metrics. -->
+    <artifactId>google-cloud-bigtable-stats</artifactId>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>google-cloud-bigtable-deps-bom</artifactId>
+                <version>2.2.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-impl</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>false</shadedArtifactAttached>
+                            <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+                            <artifactSet>
+                                <includes>
+                                    <include>io.opencensus:*</include>
+                                </includes>
+                            </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>io.opencensus</pattern>
+                                    <shadedPattern>
+                                        com.google.bigtable.repackaged.io.opencensus
+                                    </shadedPattern>
+                                </relocation>
+                            </relocations>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- ignore the warning for unused dependencies before we add the metrics with shaded OpenCensus -->
```

Comment:
this comment is incorrect ... while this artifact exists, we will have to ignore unused deps ... the dep check is only with the module